authors/apps/articles/tests/test_rate_article.py

```diff
@@ -0,0 +1,159 @@
+from django.test import TestCase
+from rest_framework import status
+from rest_framework.test import APIClient
+from django.urls import reverse
+from authors.apps.articles.models import Article, Ratings
+from authors.apps.articles.tests.test_data import (
+    valid_article, valid_article_2,
+    valid_rating, slug, invalid_rating, valid_rating_2, 
+    invalid_rating2, invalid_integer, 
+    signup_and_login_user
+    )
+from authors.apps.authentication.models import User
+from authors.apps.authentication.tests.test_data import (
+    valid_user, valid_user2
+    )
+from authors.apps.authentication.tests.app_urls import (
+    register_url, login_url, activate_url, user_action_url
+    )
+
+
+class RateArticleAPIViewTestCase(TestCase):
+
+    def setUp(self):
+        self.client = APIClient()
+
+        response = self.client.post(
+            register_url, {"user": valid_user}, format="json")
+
+        self.token = response.data["data"]["token"]
+
+        self.client.get(f"{activate_url}?token={self.token}", format='json')
+
+        response = self.client.post(
```

Comment:
Hey @NicholusMuwonge, I do not see where the use of this response. Please share where it is being used.

Reply:
I have changed the format, to the one in signup and login user method